packages/cli/src/commands/config/set.ts

```diff
@@ -10,19 +10,25 @@ export default class Set extends LocalCommand {
     // Overrides base command node flag.
     node: flags.string({
       char: 'n',
-      required: true,
+      required: false,
       description: 'URL of the node to run commands against',
       default: 'http://localhost:8545',
     }),
+    gasCurrency: BaseCommand.flags.gasCurrency,
   }
 
   static examples = ['set  --node ws://localhost:2500', 'set  --node <geth-location>/geth.ipc']
 
   async run() {
     const res = this.parse(Set)
-    const config: CeloConfig = {
-      nodeUrl: res.flags.node,
-    }
-    await writeConfig(this.config.configDir, config)
+    const curr = readConfig(this.config.configDir)
+    const node = res.flags.node ?? curr.node
```

Comment:
Double check this.
Node has a `default: 'http://localhost:8545'` so that value will always show the default.
I can't really remember but I think that something like this should help:
```
        const hasNode = res.raw.some(
          (value) => (value as any).flag === 'node'
        )
        const node = hasNode ? res.flags.node : curr.node
```

Reply:
I'm checking if an override is provided but defaulting to the config (localhost)
I have tested this with setting and getting alfajores forno node url